packages/dso-toolkit/src/components/form/form.args.ts

```diff
@@ -10,7 +11,6 @@ export interface FormArgs {
 
 export const formArgTypes: ArgTypes<FormArgs> = {
   legend: {
-    defaultValue: "Formulier",
```

Comment:
Goed gevonden! Hoe lang hebben we die deprecated warning wel niet in de console voorbij zien komen.. 🤪